app/models/spree/payment_method_decorator.rb

```diff
@@ -1,8 +1,6 @@
 Spree::PaymentMethod.class_eval do
   include Spree::Core::CalculatedAdjustments
 
-  Spree::PaymentMethod::DISPLAY = [:both, :front_end, :back_end]
```

Comment:
In the latest version of 2.0 this is still defined in Spree. See by yourself: https://github.com/openfoodfoundation/spree/blob/2-0-stable/core/app/models/spree/payment_method.rb#L4